packages/shared_preferences/shared_preferences/example/integration_test/shared_preferences_test.dart

```diff
@@ -4,30 +4,27 @@
 
 import 'dart:async';
 import 'dart:io';
+
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:shared_preferences/shared_preferences.dart';
 import 'package:integration_test/integration_test.dart';
+import 'package:shared_preferences/shared_preferences.dart';
 
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
   group('$SharedPreferences', () {
-    const Map<String, dynamic> kTestValues = <String, dynamic>{
-      'flutter.String': 'hello world',
-      'flutter.bool': true,
-      'flutter.int': 42,
-      'flutter.double': 3.14159,
-      'flutter.List': <String>['foo', 'bar'],
-    };
+    const String testString = 'hello world';
+    const bool testBool = true;
+    const int testInt = 42;
+    const double testDouble = 3.14159;
+    const List<String> testList = <String>['foo', 'bar'];
 
-    const Map<String, dynamic> kTestValues2 = <String, dynamic>{
-      'flutter.String': 'goodbye world',
-      'flutter.bool': false,
-      'flutter.int': 1337,
-      'flutter.double': 2.71828,
-      'flutter.List': <String>['baz', 'quox'],
-    };
+    const String testString2 = 'goodbye world';
+    const bool testBool2 = false;
+    const int testInt2 = 1337;
+    const double testDouble2 = 2.71828;
+    const List<String> testList2 = <String>['baz', 'quox'];
```

Comment:
I rewrote this because almost all of the code in this test requires a `Foo` (e.g., `setString` requires a `String`), so the old code when fixed naively became `kTestValues[someKey]! as Foo` everywhere, and it was a mess.